modules/sievefilters/modules.php

```diff
@@ -1274,7 +1274,7 @@ protected function output() {
 class Hm_Handler_process_enable_sieve_filter_setting extends Hm_Handler_Module {
     public function process() {
         function sieve_enabled_callback($val) { return $val; }
-        process_site_setting('enable_sieve_filter', $this, 'sieve_enabled_callback', true, true);
+        process_site_setting('enable_sieve_filter', $this, 'sieve_enabled_callback', false, true);
```

Comment:
The default value to use if callback is not submitted should be true not false here.